index.html

```diff
@@ -22,10 +23,41 @@
     </header>
     <main>
         <section>
-            <h1>Hello To Arduino Day</h1>
+            <h1>Welcome To Arduino Day</h1>
         </section>
     </main>
 
+    <div style="color: var(--blue)" class="register">
```

Comment:
This div is open but not see where is closed, & HTML semantic wrong here should be the tag `<section>`  what shoul be used, & inside of the `<main>` tag